packages/core/src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -134,6 +134,19 @@ export class AbstractQueryInterfaceTypeScript {
     return schemaNames.flatMap((value: any) => (value.schema_name ? value.schema_name : value));
   }
 
+  /**
+   * Returns a promise that will resolve to true if the table exists in the database, false otherwise.
+   *
+   * @param tableName - The name of the table
```

Comment:
```suggestion
   * Returns a promise that will resolve to true if the table or model exists in the database, false otherwise.
   *
   * @param tableName - The name of the table or model
```